libs/core/src/lib/select/select.component.scss

```diff
@@ -18,3 +17,14 @@
         max-width: inherit;
     }
 }
+
+/**
+   Dirty hack to enforce z-index of CDK's overlay class to be higher then Dialog z-index
+ */
```

Comment:
There should be a `*` in front of the text. Same is valid for the prev type doc comment